lib/Twig/Loader/Filesystem.php

```diff
@@ -302,4 +302,34 @@ private function isAbsolutePath($file)
             || null !== parse_url($file, PHP_URL_SCHEME)
         ;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getTemplateFreshness(Twig_CacheInterface $cache)
+    {
+        $cacheKey = $cache->generateKey(
+            md5(serialize($this->cache)),
```

Comment:
As the order of include template change the rendering of a template, yes the order is important